client/components/CompanyProfile/CompanyProfile.tsx

```diff
@@ -115,6 +117,14 @@ export default withAC(
               </>
             )}
           </Col>
+          <Col md="2" className="float-right text-right">
+            <small>Partner Team</small>
+            <PartnerTeamDropdown
+              partnerTeams={Object.values(Team)}
```

Comment:
Instead of passing this through props why not just import the Team enum into the dropdown file?

Reply:
Looks like you already imported Team into PartnerTeamDropdown.tsx